pkg/config/release.go

```diff
@@ -213,7 +215,7 @@ func GetChangedRegistrySteps(path, baseRev string, graph registry.NodeByName) ([
 		return changes, err
 	}
 	for _, c := range revChanges {
-		if filepath.Ext(c) == ".yaml" || strings.HasSuffix(c, load.CommandsSuffix) {
+		if filepath.Ext(c) == ".yaml" || strings.Contains(c, load.CommandsSuffix) {
```

Comment:
This is too non-specific IMO and could result in an error if a step contains a `-commands` string in it since this might then try to load the metadata JSON for the step. I think something like `strings.HasSuffix(c, fmt.Sprintf("%s%s", load.CommandsSuffix, filepath.Ext(c)))` would be better.

Reply:
Picked up your suggested alternative in c75863a18 -> c47694dbb.